src/grammar.json

```diff
@@ -738,8 +738,124 @@
     ],
 
     "card": [
```

Comment:
Any one in particular? I looked at each of them and thought all of them would be pretty good if they randomly said "a HP coffee-stained Wall of Wood" for instance.